### ISSUE TITLE
Compiling ripgrep_all fails due to missing cachedir 0.1.1

### ISSUE THREAD

**Issue by mcamou** [phiresky/ripgrep-all#156]
```
$ cargo install ripgrep_all
    Updating crates.io index
  Installing ripgrep_all v0.9.6
error: failed to compile `ripgrep_all v0.9.6`, intermediate artifacts can be found at `/tmp/cargo-installqNIgK9`

Caused by:
  failed to select a version for the requirement `cachedir = "^0.1.1"`
  candidate versions found which didn't match: 0.3.0, 0.2.0
  location searched: crates.io index
  required by package `ripgrep_all v0.9.6`

$ rustc --version
rustc 1.67.0 (fc594f156 2023-01-24)

$ cargo version
cargo 1.67.0 (8ecd4f20a 2023-01-10)
```

**Comment by bdaase** [comment]
See https://github.com/phiresky/ripgrep-all/issues/96#issuecomment-841828835, you can use `cargo install ripgrep_all --git https://github.com/phiresky/ripgrep-all.git` for now.

**Comment by phiresky** [comment]
fixed in 1.0.0-alpha.4
